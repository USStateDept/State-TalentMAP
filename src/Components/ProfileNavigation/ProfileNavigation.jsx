import { Link } from 'react-router-dom';

const ProfileNavigation = () => (
  <div className="usa-grid-full">
    <Link to="/profile">
      Profile home
    </Link>
    <br />
    <Link to="/profile/favorites">
      Favorites
    </Link>
    <br />
    <Link to="/profile/searches">
      Saved Searches
    </Link>
    <br />
    <Link to="/profile/bidlist">
      Bid List
    </Link>
  </div>
);

export default ProfileNavigation;
