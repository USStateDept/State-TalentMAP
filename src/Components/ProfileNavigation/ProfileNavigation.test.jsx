import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ProfileNavigation from './ProfileNavigation';

describe('ProfileNavigationComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <ProfileNavigation />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <ProfileNavigation />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
