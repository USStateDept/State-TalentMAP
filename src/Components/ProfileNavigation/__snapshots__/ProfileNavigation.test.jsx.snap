// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileNavigationComponent matches snapshot 1`] = `
<div
  className="usa-grid-full"
>
  <Link
    replace={false}
    to="/profile"
  >
    Profile home
  </Link>
  <br />
  <Link
    replace={false}
    to="/profile/favorites"
  >
    Favorites
  </Link>
  <br />
  <Link
    replace={false}
    to="/profile/searches"
  >
    Saved Searches
  </Link>
  <br />
  <Link
    replace={false}
    to="/profile/bidlist"
  >
    Bid List
  </Link>
</div>
`;
